### ISSUE TITLE
GHA Error: java.lang.Throwable: Problems found loading plugins:

### ISSUE THREAD

**Issue by bryanlb** [JetBrains/writerside-github-action#16]
As of this morning the Github action has started throwing errors and fatal exiting the docker container due to a missing repository search plugin.

This appears to be related to the newest release.

```
CompileCommand: exclude com/intellij/openapi/vfs/impl/FilePartNodeRoot.trieDescend bool exclude = true
Apr 25, 2024 5:24:37 PM java.util.prefs.FileSystemPreferences$1 run
INFO: Created user preferences directory.
2024-04-25 17:24:37,857 [    371] SEVERE - #c.i.i.p.PluginManager - Problems found loading plugins:
  The JetBrains Repository Search (id=org.jetbrains.idea.reposearch, path=/opt/builder/plugins/repository-search) plugin Plugin 'JetBrains Repository Search' requires plugin 'org.jetbrains.idea.maven.model' to be installed
java.lang.Throwable: Problems found loading plugins:
  The JetBrains Repository Search (id=org.jetbrains.idea.reposearch, path=/opt/builder/plugins/repository-search) plugin Plugin 'JetBrains Repository Search' requires plugin 'org.jetbrains.idea.maven.model' to be installed
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:376)
	at com.intellij.ide.plugins.PluginManagerCore.preparePluginErrors(PluginManagerCore.kt:323)
	at com.intellij.ide.plugins.PluginManagerCore.initializePlugins(PluginManagerCore.kt:619)
	at com.intellij.ide.plugins.PluginManagerCore$initializeAndSetPlugins$2.invokeSuspend(PluginManagerCore.kt:814)
	at com.intellij.ide.plugins.PluginManagerCore$initializeAndSetPlugins$2.invoke(PluginManagerCore.kt)
	at com.intellij.ide.plugins.PluginManagerCore$initializeAndSetPlugins$2.invoke(PluginManagerCore.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:78)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:167)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.diagnostic.telemetry.impl.TracerKt.span(tracer.kt:56)
	at com.intellij.idea.Main$startApp$2$1$1.span(Main.kt:90)
	at com.intellij.diagnostic.CoroutineTracerShim.span$default(CoroutineTracerShim.kt:31)
	at com.intellij.ide.plugins.PluginManagerCore.initializeAndSetPlugins$intellij_platform_core_impl(PluginManagerCore.kt:813)
	at com.intellij.ide.plugins.PluginDescriptorLoader$scheduleLoading$pluginSetDeferred$1.invokeSuspend(PluginDescriptorLoader.kt:409)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
2024-04-25 17:24:37,879 [    393] SEVERE - #c.i.i.p.PluginManager - IntelliJ IDEA 2023.3.1  Build #IC-233.11799.300
2024-04-25 17:24:37,882 [    396] SEVERE - #c.i.i.p.PluginManager - JDK: 17.0.9; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2024-04-25 17:24:37,883 [    397] SEVERE - #c.i.i.p.PluginManager - OS: Linux
2024-04-25 17:24:38,420 [    934]   WARN - #c.i.s.ComponentManagerImpl - `preload=true` must be used only for core services (service=com.jetbrains.rdserver.statistics.BackendStatisticsManager, plugin=com.jetbrains.codeWithMe)
2024-04-25 17:24:39,099 [   1613]   WARN - #c.i.s.ComponentManagerImpl - com.intellij.psi.search.FilenameIndex initializer requests com.intellij.ide.plugins.PluginUtil instance
Cannot parse CLI arguments: Unrecognized option: -time
Test existing artifacts
Artifacts not found
Error: The process '/usr/bin/docker' failed with exit code 1
```
via https://github.com/slackhq/astra/actions/runs/8836809770/job/24264216610
via https://github.com/slackhq/astra/actions/runs/8836942753/job/24264636365

I'm using the workflow from the recommended template - https://github.com/slackhq/astra/blob/master/.github/workflows/docs.yml

**Comment by bryanlb** [comment]
I believe this may be the same issue as is referred to in #15

**Comment by bryanlb** [comment]
This was resolved in #15 with a rollback of the most recent release. Thanks @siarb!
